src/index.html

```diff
@@ -11,11 +11,22 @@
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
   <script src="assets/Chart.bundle.min.js"></script>
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-
+  <!-- Meta data for search engines -->
   <meta name="description"
     content="Susper is a decentral Search Engine that uses the peer to peer system
     'YaCy' and 'Apache Solr' to crawl and index search results.">
-
+  <!-- @0x48piraj :: Basic meta data for ranking high in google search, and making large tiles in Twitter when susper.com is mentioned.  -->
```

Comment:
No need of username here. @0x48piraj

Reply:
It's just a comment. I read that in huge codebases, devs comment the feature or code block they developed, so did I :blush:

Reply:
But here we don't need it.